src/Export/ProductService.php

```diff
@@ -288,4 +338,60 @@ protected function assignChildrenOrSiblings(ProductEntity $product): void
         $children = $this->getChildrenOrSiblings($product);
         $product->setChildren($children);
     }
+
+    private function getParentByMainProduct(ProductEntity $product): ProductEntity
```

Comment:
Change all `private` methods in this class to be `protected`, otherwise extensions can not call them.